docs/management/persistence.md

```diff
@@ -259,42 +259,35 @@ and starts appending new data into the new file.
 
 ### How I can switch to AOF, if I'm currently using dump.rdb snapshots?
 
-There is a different procedure to do this in version 2.0 and later versions, as you
-can guess it's simpler since Redis 2.2 and does not require a restart at all.
+If you want to enable AOF in a server that is currently using RDB snapshots,
+you need to convert the data interactively using `redis-cli`.
 
-**Redis >= 2.2**
+**IMPORTANT:** not following this procedure (e.g. just changing the config and
+restarting the server) can result in data loss!
+
+The following steps require at least Redis 2.2:
 
 * Make a backup of your latest dump.rdb file.
 * Transfer this backup to a safe place.
-* Issue the following two commands:
-* `redis-cli config set appendonly yes`
-* `redis-cli config set save ""`
-* Make sure your database contains the same number of keys it contained.
+* Enable AOF: `redis-cli config set appendonly yes`
+* Optionally disable RDB: `redis-cli config set save ""`
+* Make sure your database contains the same number of keys it contained
+  previously.
 * Make sure writes are appended to the append only file correctly.
+* Update your `redis.conf` and ensure that it matches the config above.
 
 The first CONFIG command enables the Append Only File persistence.
 
-The second CONFIG command is used to turn off snapshotting persistence. This is optional, if you wish you can take both the persistence methods enabled.
+The second CONFIG command is used to turn off snapshotting persistence. This is
+optional, if you wish you can take both the persistence methods enabled.
 
 **IMPORTANT:** remember to edit your redis.conf to turn on the AOF, otherwise
 when you restart the server the configuration changes will be lost and the
-server will start again with the old configuration.
-
-**Redis 2.0**
-
-* Make a backup of your latest dump.rdb file.
-* Transfer this backup into a safe place.
-* Stop all the writes against the database!
-* Issue a `redis-cli BGREWRITEAOF`. This will create the append only file.
-* Stop the server when Redis finished generating the AOF dump.
-* Edit redis.conf end enable append only file persistence.
-* Restart the server.
-* Make sure that your database contains the same number of keys it contained before the switch.
-* Make sure that writes are appended to the append only file correctly.
```

Comment:
Delete by mistake?

Reply:
Not really, I still see some users still using redis 3.x.

Reply:
Redis 3 was released 4 years after 2.0, and the current instructions will work from 2.2 onwards. But sure, I can re-add these instructions.